_layouts/base.html

```diff
@@ -2,9 +2,9 @@
 layout: default
 ---
 {% assign home = "" %}
-{% if site.github.environment == "dotcom" %}
-    {% assign home = site.github.url %}
-{% endif %}
+<!-- {% if site.github.environment == "dotcom" %} -->
```

Comment:
I believe if you comment this out, then the site will not work when served from a local Jekyll running on your machine. That's why I needed the conditional.